plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/internal/ExternalProjectCacheLoader.java

```diff
@@ -61,7 +61,7 @@
 				return Optional.of(Tuples.create(ppp, projectDescription));
 			}
 		} catch (N4JSBrokenProjectException e) {
-			LOGGER.error("Failed to obtain project description for external library.", e);
+			LOGGER.error("Failed to obtain project description for external library.");
```

Comment:
why no `e` ?

Reply:
It will print the stacktrace, but this case is handled by the IDE. So this stacktrace is maybe confusing.